packages/react/src/use-assistant.ui.test.tsx

```diff
@@ -7,27 +7,29 @@ import '@testing-library/jest-dom/vitest';
 import { cleanup, findByText, render, screen } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import { useAssistant } from './use-assistant';
+import { firstRun } from './samples';
```

Comment:
since we use samples, maybe move to a test subfolder and rename to `test-samples`

Reply:
done!